packages/discussions/src/types.ts

```diff
@@ -705,6 +705,40 @@ export interface ISearchChannelsOptions extends IHubRequestOptions {
   params?: ISearchChannels;
 }
 
+/**
+ * permission object that will populate ACL interface
+ *
+ * @export
+ * @interface IPermission
+ */
+export interface IPermission {
+  role: string;
```

Comment:
Should the role just be a primitive string?  What about a distinct set of string values or enums?

Reply:
Yea, good idea to strictly constrain that.